### PR TITLE
Use other method to enumerate contents, which causing crash(maybe only occurred for archved build)

### PR DESCRIPTION
**Pull request by mono0926**
- Fix https://github.com/onevcat/Kingfisher/issues/619

### CODE REVIEW

**Comment by mono0926** [comment]
I don't know why `enumerator(at:includingPropertiesForKeys:options:errorHandler:)` caused crash but   the crash resolved by using `contentsOfDirectory(at:includingPropertiesForKeys:options:)` instead it.
Anyway, the latter is simpler, I think.

I thought these problems are involved, but it seems to be other problems.

- [[SR-2690] Bad code generated for FileManager.enumerator(at:includingPropertiesForKeys:options:errorHandler:) - Swift](https://bugs.swift.org/browse/SR-2690)
- [[SR-2872] Crash when FileManager's enumerate(at:...) tries to report an error - Swift](https://bugs.swift.org/browse/SR-2872)

File: Sources/ImageCache.swift [hunk 1]
```diff
@@ -481,42 +481,38 @@ open class ImageCache {
...
-                        urlsToDelete.append(fileUrl)
-                        continue
-                    }
 
-                    if let fileSize = resourceValues.totalFileAllocatedSize {
-                        diskCacheSize += UInt(fileSize)
-                        if !onlyForCacheSize {
-                            cachedFiles[fileUrl] = resourceValues
-                        }
+        for fileUrl in (try? fileManager.contentsOfDirectory(at: diskCacheURL, includingPropertiesForKeys: Array(resourceKeys), options: FileManager.DirectoryEnumerationOptions.skipsHiddenFiles)) ?? [] {
```
Comment (onevcat) [inline Sources/ImageCache.swift]: Sorry for the original bad code, but `FileManager.DirectoryEnumerationOptions.skipsHiddenFiles` could be simplified to `.skipsHiddenFiles`.

Comment (mono0926) [inline Sources/ImageCache.swift]: @onevcat 

Okay, I corrected and rebased: https://github.com/onevcat/Kingfisher/pull/620/commits/33b3f28386a76740cc099d28e4bb5c51b2587324

**Comment by mono0926** [comment]
@onevcat 

Thanks, if it is okay, merge please :octocat:

**Comment by onevcat** [comment]
Great!

**Pull request merged**